src/components/Groups/Group/index.js

```diff
@@ -63,9 +67,35 @@ const Group = props => {
     getActiveMembers()
   }, [members, profiles, setActiveMembers, teams])
 
+  useEffect(() => {
+    // Get channel posts
+    const fetchPosts = async () => {
+      if (channel && channel.id) {
+        const channelPosts = await getPosts(channel.id)
+        setPosts(channelPosts.data.posts)
+      }
+    }
+    fetchPosts()
+  }, [channel, getPosts])
+
+  // get unread posts for channel
+  useEffect(() => {
+    const getUnreadPosts = async () => {
+      if (unreadCount && unreadCount > 0 && posts) {
+        const beginIndex = Object.keys(posts).length - unreadCount
+        const getUnreadMessages = Object.values(posts)
+          .sort((p1, p2) => p1.create_at - p2.create_at)
+          .slice(beginIndex)
+        if (getUnreadMessages)
+          setUnreadPosts(getUnreadMessages.filter(p => p.type === '').length)
```

Comment:
what is this comparison doing? `p.type === ''` 🤔

Reply:
If the message is a "regular" message its type is an empty string. So this takes only the "real" messages and ignores the ones with type system-something.

Reply:
Ah, I see! :)